CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib

```diff
@@ -25,6 +25,845 @@ @online{SKA
+      reportNumber   = "FERMILAB-PUB-13-111-E",
+      SLACcitation   = "%%CITATION = ARXIV:1305.5199;%%"
+}
+
+@article{Acciarri:2015uup,
+      author         = "Acciarri, R. and others",
+      title          = "{Long-Baseline Neutrino Facility (LBNF) and Deep
+                        Underground Neutrino Experiment (DUNE)}",
+      collaboration  = "DUNE",
+      year           = "2015",
+      eprint         = "1512.06148",
+      archivePrefix  = "arXiv",
+      primaryClass   = "physics.ins-det",
+      reportNumber   = "FERMILAB-DESIGN-2016-02",
+      SLACcitation   = "%%CITATION = ARXIV:1512.06148;%%"
+}
+
+@article{Ferrari:2005zk,
+      author         = "Ferrari, Alfredo and Sala, Paola R. and Fasso, Alberto
+                        and Ranft, Johannes",
+      title          = "{FLUKA: A multi-particle transport code (Program version
+                        2005)}",
+      year           = "2005",
+      reportNumber   = "CERN-2005-010, SLAC-R-773, INFN-TC-05-11",
+      SLACcitation   = "%%CITATION = CERN-2005-010;%%"
+}
+
+@article{1742-6596-396-5-052035,
+  author={Marek Gayer and John Apostolakis and Gabriele Cosmo and Andrei Gheata and Jean-Marie Guyader and Tatiana Nikitina},
+  title={New software library of geometrical primitives for modeling of solids used in Monte Carlo detector simulations},
+  journal={Journal of Physics: Conference Series},
+  volume={396},
+  number={5},
+  pages={052035},
+  url={http://stacks.iop.org/1742-6596/396/i=5/a=052035},
+  year={2012},
+}
+
+@article{1742-6596-608-1-012023,
+  author={J Apostolakis and M Bandieramonte and G Bitzes and R Brun and P Canal and F Carminati and G Cosmo and J C De Fine Licht and L
+Duhem and V D Elvira and A Gheata and S Y Jun and G Lima and T Nikitina and M Novak and R Sehgal and O Shadura and S Wenzel},
+  title={Towards a high performance geometry library for particle-detector simulations},
+  journal={Journal of Physics: Conference Series},
+  volume={608},
+  number={1},
+  pages={012023},
+  url={http://stacks.iop.org/1742-6596/608/i=1/a=012023},
+  year={2015},
+}
+
+@ARTICLE{1710291, 
+  author={R. Chytracek and J. Mccormick and W. Pokorski and G. Santin}, 
+  journal={IEEE Transactions on Nuclear Science}, 
+  title={Geometry Description Markup Language for Physics Simulation and Analysis Applications}, 
+  year={2006}, 
+  volume={53}, 
+  number={5}, 
+  pages={2892-2896}, 
+  keywords={XML;computational geometry;data visualisation;digital simulation;physics computing;position sensitive particle detectors;CMS;GDML;Geant4;ROOT;XML-based language;application-independent persistent format;compact muon spectrometer;data visualisation;detector geometry;geometry description markup language;physics measurements;physics simulation;Analytical models;Detectors;Engines;Geometry;Helium;Markup languages;Physics;Solid modeling;Visualization;XML;Detector;GDML;Geant4;geometry;root;simulation}, 
+  doi={10.1109/TNS.2006.881062}, 
+  ISSN={0018-9499}, 
+  month={Oct},
```

Comment:
Biber advises changing this to

`month=Oct` (no braces)

See https://tex.stackexchange.com/questions/330179/month-field-not-an-integer-will-not-sort-properly.